### DISCUSSION TITLE
Add option for migration rollback

### DISCUSSION THREAD

**Discussion by germgerm** [JetBrains/rider-efcore#167] [Suggestions]
One less thing to have to drop out to the command line for ?

**Comment by seclerp** [comment]
Hi @germgerm, which particular EF command you would like to replace with UI?

**Comment by germgerm** [comment]
could display a list of all migrations and allow selecting one to rollback to, with option to remove code migrations

rollback migrations : **dotnet ef database update _MigrationToRollbackTo_ --context _MyContext_**
optionally remove code migration(s) : **dotnet ef migrations remove --context _MyContext_**

**Reply by seclerp** [reply]
The plugin already has UI for "Update Database" and "Remove Last Migration", are you missing some functionality pieces from them?

**Reply by seclerp** [reply]
You could easily select the migration you want to rollback into

![image](https://github.com/seclerp/rider-efcore/assets/20597871/d8e6da10-8f22-4028-804f-2c46707a898b)

**Comment by germgerm** [comment]
Sorry I just checked the update option, I guess I never used it  and didn't realize it had the option to select the migration - was assuming it was just for applying the latest.
Thanks for this plugin, it has been great since I've migrating everything to core - replaced a lot of scripts and run configs :)
